lib/netsnmp/client.rb

```diff
@@ -135,6 +135,17 @@ def set(*oid_opts)
       values.size > 1 ? values : values.first
     end
 
+    # Perform a SNMP INFORM Request
+    #
+    # @see {NETSNMP::Varbind#new}
+    #
+    def inform(oid_opts)
+      request = @session.build_pdu(:inform, oid_opts)
+      response = handle_retries { @session.send(request) }
+      yield response if block_given? 
+      response
```

Comment:
What do you think about keeping here the same pattern as the other methods here? : 

```ruby
values = response.varbinds.map { |v| [v.oid, v.value] }
 values.size > 1 ? values : values.first
```

unless you had your reasons not to.